assets/app/view/game/part/label.rb

```diff
@@ -17,7 +17,7 @@ class Label < Base
             y: 0,
           },
           pointy: {
-            region_weights: { [5, 6] => 1.0, [7, 12] => 0.25 },
+            region_weights: { [0, 1] => 1.0, [2, 6] => 0.25 },
```

Comment:
did you validate how this affects other games?

Reply:
I have checked all tiles and map tiles of all games with pointy, same applies to the other change in this commit.

Reply:
It messed up 1866, because there are icons on the map there as well. All the icons and the labels are on the same place.

Reply:
I definitely thought I checked, sorry about that. I do however believe the old regions were "wrong" if you consider which region denotes which part of the tile. Will take a look today to see if I can fix it.

Reply:
I've submitted a new pull request to fix the issue.